@here/harp-materials/lib/Utils.ts

```diff
@@ -57,10 +59,29 @@ export function enforceBlending(
     material.forcedBlending = true;
 }
 
+/**
+ * Enable alpha blending using THREE.CustomBlending setup.
+ *
+ * Function enables blending using one of predefined modes, for both color and alpha components:
+ * - Src: [[THREE.SrcAlphaFactor]], Dst: [[THREE.OneMinusSrcAlphaFactor]]
+ * - Src: [[THREE.OneFactor]], Dst: [[THREE.OneMinusSrcAlphaFactor]]
+ * The second blending equation is used when [[THREE.Material.premultipliedAlpha]] is enabled
+ * for this material.
+ *
+ * @param material The material or material parameters to modify.
+ * @param isUpdate Set to true if you want to notify about material change, required only if you
+ * change blend mode after material has been created and passed to WebGL, it does not have meaning
+ * when [[THREE.ShaderMaterialParameters]] are passed into function. By default set to [[false]].
+ */
 export function enableBlending(
-    material: (THREE.Material | THREE.ShaderMaterialParameters) & ForcedBlending
+    material: (THREE.Material | THREE.ShaderMaterialParameters) & ForcedBlending,
+    isUpdate: boolean = false
 ) {
-    if (material.transparent || material.forcedBlending) {
+    if (
+        material.transparent ||
+        material.forcedBlending ||
+        material.blending === THREE.CustomBlending
```

Comment:
The clue of this fix is here. Do not force material update when nothing changed.